modules/exploits/linux/http/tr069_ntpserver_cmdinject.rb

```diff
@@ -0,0 +1,146 @@
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Eir D1000 Modem CWMP Exploit POC',
```

Comment:
Absolutely not; easy order of business is to retitle, re-describe, and re-reference the thing.